openssl-dynamic/src/main/c/sslutils.c

```diff
@@ -555,8 +554,7 @@ int SSL_CTX_use_certificate_chain(SSL_CTX *ctx, const char *file,
     return n;
 }
 
-int SSL_CTX_use_certificate_chain_bio(SSL_CTX *ctx, BIO *bio,
-                                  int skipfirst)
+static int SSL_CTX_setup_certs(SSL_CTX *ctx, BIO *bio, int skipfirst, int ca)
```

Comment:
good question. I just used what we used in other methods. So keep it for now as it is ?